### PR TITLE
updated Luminus to 2.x

### PR DESCRIPTION
**Pull request by yogthos**
This pull request updates the Luminus framework benchmark test to the latest version of the framework.

### CODE REVIEW

**Comment by yogthos** [comment]
any chance I could get a hand figuring out what broke, I tested locally without problems and I can't see what went wrong on travis

File: frameworks/Clojure/luminus/hello/src/hello/routes/home.clj [hunk 1]
```diff
@@ -62,14 +39,11 @@
...
    :body "Hello, World!"})
 
-
 (defroutes home-routes
   (GET "/"                [] "Hello, World!")
   (GET "/plaintext"       [] plaintext)
   (GET "/json"            [] json-serialization)
-  (GET "/db"              [] single-query-test)
+  (GET "/db"              [] (single-query-test))
   (GET "/db/:queries"     [queries] (multiple-query-test queries))
```
Comment (hamiltont) [inline frameworks/Clojure/luminus/hello/src/hello/routes/home.clj]: Are you sure the query url is correct in this section? In the benchmark_config I see `query_url": "/queries/",` but I don't see a `/queries` url mentioned here

File: frameworks/Clojure/luminus/hello/src/hello/routes/home.clj [hunk 2]
```diff
@@ -62,14 +39,11 @@
...
-  (GET "/dbraw"           [] single-query-test-raw)
-  (GET "/dbraw/:queries"  [queries] (multiple-query-test-raw queries))
-  (GET "/fortune"         [] fortunes)
+  (GET "/fortune"         [] (fortunes))
```
Comment (hamiltont) [inline frameworks/Clojure/luminus/hello/src/hello/routes/home.clj]: In benchmark_config I think you set this to be `/fortunes`, not `/fortune`

Comment (yogthos) [inline frameworks/Clojure/luminus/hello/src/hello/routes/home.clj]: ah thanks for the eagle eye! :)

**Comment by yogthos** [comment]
@hamiltont thanks for the help!

**Comment by hamiltont** [comment]
FYI, we don't require the charset header (e.g. those `charset=utf-8` scattered throughout). By leaving it in you are using more bytes per request than the other frameworks that don't send this header info. That's what the WARN in the lastest travis-ci verification results is trying to tell you (our feedback message for this particular warning is still a bit wonky). I'm happy to merge as-is, but if you want to update this it's a good time. Let me know

**Comment by yogthos** [comment]
thanks just removed that

**Comment by yogthos** [comment]
awesome!

**Comment by hamiltont** [comment]
:+1:

**Pull request merged**